src/api/buddies.ts

```diff
@@ -17,21 +17,45 @@ const buddyType = t.intersection([
   t.partial({ status: t.literal('banned') }),
 ]);
 
-export type BanActions = 'Ban' | 'Unban';
-export type BanStatuses = 'Banned' | 'NotBanned';
+const buddiesType = t.strict({ resources: t.array(buddyType) });
+
+export type BanActions = 'Ban' | 'Unban' | 'Delete';
+export type BanStatuses = 'Banned' | 'NotBanned' | 'Deleted';
+export type BanStatusStrings = 'banned' | 'ok' | 'deleted';
```

Comment:
Very nitpicky, but i think it would be better to be called _BanStatusString_ in singular form.

Reply:
I agree, but the other types' names are also in plural form. So for consistency, I would leave it like that or change all of them to singular form.